tools/wake-format/emitter.cpp

```diff
@@ -978,7 +979,47 @@ wcl::doc Emitter::walk_ideq(ctx_t ctx, CSTElement node) {
 
 wcl::doc Emitter::walk_if(ctx_t ctx, CSTElement node) {
   MEMO(ctx, node);
-  MEMO_RET(walk_placeholder(ctx, node));
+  assert(node.id() == CST_IF);
+
+  auto fits_no_nl =
```

Comment:
I didn't see a test for this case up above.

Reply:
It was already in there, I only noticed to fix this because there was a regression. I'll add an explicit one though